client-mvp-04/src/components/createAccount/createAccountView.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import './createAccount.scss';
+import Button from '../common/button/button';
+import Title from '../common/title/title';
+import Input from '../common/input/input';
+import ErrorMessage from '../common/errorMessage/errorMessage';
+import RedirectLink from '../common/link/link';
+
+
+const CreateAccountView = ({
+    handleSubmitForm,
+    handleInputChange,
+    isDisabled,
+    isEmailValid,
+    errorMsgInvalidEmail,
+    errorMsgRegisteredEmail,
+}) => {
+    return (
+        <section data-testid="createAccount" className="createAccount-container">
```

Comment:
All classes and data-testid should be named using kebab-case style: `create-account-container`